clients/build.rb

```diff
@@ -23,7 +23,7 @@ def clone(lang)
   Dir.chdir 'tmp'
   ldir = "fn_#{lang}"
   if !Dir.exist? ldir
-    cmd = "git clone https://github.com/fnproject/#{ldir}"
```

Comment:
why'd you change this?

Reply:
Personally, I don't like login/passwords, if I can do simpler thing nothing would stop me, that's why relying on SSH authorisation/authentication is much more preferable, because you don't have to keep in mind your passwords.
Even in case if you do this from your work VM you might just configure one more SSH key for this type of things.